### PR TITLE
Add --docker_only flag to enable tracking for only docker containers & root.

### PR DESCRIPTION
**Pull request by rjnagal**
This reduces unnecessary load on the system and also cleans up the UI clutter.
Currently defaulted to false.

### CODE REVIEW

File: manager/manager.go [hunk 1]
```diff
@@ -619,6 +622,22 @@ func (m *manager) GetVersionInfo() (*info.VersionInfo, error) {
...
 
+func (m *manager) Exists(containerName string) bool {
+	m.containersLock.Lock()
+	defer m.containersLock.Unlock()
+
+	namespacedName := namespacedContainerName{
+		Name: containerName,
+	}
+
+	// Check that the container didn't already exist.
```
Comment (vmarmol) [inline manager/manager.go]: stale comment?

**Comment by vmarmol** [comment]
We'll print out each new raw container as a failure. We will also "detect" them every 60s and re-print out the failure. We need to somehow tell the manager to ignore the container. Maybe create it but don't house keep it?

File: manager/manager.go (hunk 1 above)
Comment (rjnagal) [inline manager/manager.go]: removed

**Comment by rjnagal** [comment]
Just ignoring housekeeping still shows the container in all APIs and UI. Its a query-able container that doesn't return anything. 

Uploaded a different fix that distinguish failed vs ignored containers in manager and silently fails the latter. PTAL.

**Comment by vmarmol** [comment]
LGTM, thanks @rjnagal! Merging, the e2e machines are unhappy :(

**Pull request merged**